### ISSUE TITLE
ProgressSpinner: Unable to set aria-label

### ISSUE THREAD

**Issue by akashuwtw** [primefaces/primeng#14798]
### Describe the bug

We are trying to set `aria-label` for `p-progressSpinner` but it is not getting applied as per the steps mentioned [here](https://primeng.org/progressspinner#accessibility).

### Environment

OS: Windows 10

### Angular version

16.2.7

### PrimeNG version

16.5.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.16.0

### Browser(s)

Chrome 121.0.6167.161, Edge 121.0.2277.112

### Steps to reproduce the behavior

<p-progressSpinner aria-label="Loading"></p-progressSpinner>

### Expected behavior

aria-label should be applied
